storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -140,6 +140,13 @@
     public static final String STORM_META_SERIALIZATION_DELEGATE = "storm.meta.serialization.delegate";
 
     /**
+     * A list of statistics  preparable reporter class names.
+     */
+    @NotNull
+    @isStringList
+    public static final String STORM_STATISTICS_PREPARABLE_REPORTER_PLUGIN = "storm.statistics.preparable.reporter.plugin";
```

Comment:
Not sure myself, but how about `storm.daemon.metrics.reporter.plugins` ?

Wanted to specify that this was for daemon metrics and make it plural.